docs/diagrams/UiClassDiagram.puml

```diff
@@ -33,25 +35,31 @@ UiManager -down-> "1" MainWindow
 MainWindow *-down->  "1" CommandBox
```

Comment:
Would it be better to change line 33 to `UiManager .right.|> Ui`, so that it looks less cluttered?

Currently `UiManager .left.|> Ui`:
<img width="448" alt="Screenshot 2024-10-22 at 23 01 24" src="https://github.com/user-attachments/assets/00bac598-9f89-439f-8249-0e9e5b5e83bd">


After changing to `UiManager .right.|> Ui`:
<img width="441" alt="Screenshot 2024-10-22 at 23 00 43" src="https://github.com/user-attachments/assets/343273c6-096a-4443-a8de-6eb24055a375">

Reply:
Makes sense, I'll change that now!